gdk-pixbuf/src/auto/pixbuf.rs

```diff
@@ -72,6 +72,68 @@ impl Pixbuf {
     //    unsafe { TODO: call ffi:gdk_pixbuf_new_from_data() }
     //}
 
+    #[doc(alias = "gdk_pixbuf_new_from_file")]
+    #[doc(alias = "new_from_file")]
+    pub fn from_file(filename: impl AsRef<std::path::Path>) -> Result<Pixbuf, glib::Error> {
+        unsafe {
+            let mut error = ptr::null_mut();
+            let ret = ffi::gdk_pixbuf_new_from_file(filename.as_ref().to_glib_none().0, &mut error);
+            if error.is_null() {
+                Ok(from_glib_full(ret))
+            } else {
+                Err(from_glib_full(error))
+            }
+        }
+    }
+
+    #[doc(alias = "gdk_pixbuf_new_from_file_at_scale")]
+    #[doc(alias = "new_from_file_at_scale")]
+    pub fn from_file_at_scale(
+        filename: impl AsRef<std::path::Path>,
+        width: i32,
+        height: i32,
+        preserve_aspect_ratio: bool,
+    ) -> Result<Option<Pixbuf>, glib::Error> {
```

Comment:
It seems like the `nullable = false` annotation here does not work correctly. Any guess @bilelmoussaoui @GuillaumeGomez ?